.github/workflows/convert.yaml

```diff
@@ -0,0 +1,66 @@
+name: Publish ED-318
+
+on:
+  workflow_dispatch: {}
+
+env:
+    ED269_URL: https://data.geo.admin.ch/ch.bazl.einschraenkungen-drohnen/einschraenkungen-drohnen/einschraenkungen-drohnen_4326.json
+    ED318_FILENAME: ed318.json
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  convert-with-geospatial-utils:
+    name: Convert to ED318 using geospatial-utils
+
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout geospatial-utils
+        uses: actions/checkout@v4
+        with:
+          repository: interuss/geospatial-utils
+          submodules: 'recursive'
+
+      - name: Set permissions on file system
+        run: mkdir ./geospatial-utils/.cache && mkdir ./geospatial-utils/output && chmod -R 755 ./geospatial-utils/.cache && chmod -R 755 ./geospatial-utils/output
+
+
+      - name: Convert ED269 to ED318
+        run: ./geospatial-utils/run_locally.sh convert "${ED269_URL}" output/"${ED318_FILENAME}"
```

Comment:
Nit: The outout path seems to be relative to the script location. This is not standard, the path should be relaive to the cwd.